schema.sql

```diff
@@ -62,4 +62,12 @@ CREATE TABLE visits (
  date_of_visit DATE,
  CONSTRAINT vet_fk FOREIGN KEY(vets_id) REFERENCES vets(id),
  CONSTRAINT animals_fk FOREIGN KEY(animals_id) REFERENCES animals(id)
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+/* Create index on visits table */
+CREATE INDEX visits_animals_id_idx ON visits(id);
+CREATE INDEX visits_vets_id_idx ON visits(id);
+CREATE INDEX visits_owners_id_idx ON owners(email);
```

Comment:
For query:  `SELECT * FROM owners where email = 'owner_18327@mail.com';`
Please make sure that the first screenshot should show how much time the query takes BEFORE the improvements are introduced and it should show the usage of `Seq scan and/or Filter` in the output.